src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/TestHelpers.cs

```diff
@@ -16,13 +15,23 @@ public static class TestHelpers
 
         private static string s_tcpUrl;
         private static string s_httpUrl;
+        private static string s_tcpDupUrl;
+        private static string s_httpDupUrl;
+        private static string s_tcpStreamingUrl;
+        private static string s_httpStreamingUrl;
 
+       
         public static void SetHostAndProtocol(bool useHttp, string hostName, string appName)
         {
             UseHttp = useHttp;
```

Comment:
Are we going to test using, say, https? If so, maybe we should future-proof this rather than have this be a `bool`. Not needed for merge, just consideration.